mixer/pkg/perf/run.go

```diff
@@ -57,22 +62,24 @@ func (b *testingBWrapper) n() int {
 	return b.b.N
 }
 
-// RunInprocess executes the benchmark by launching the mixer within the same process.
-func RunInprocess(b *testing.B, setup *Setup, env *Env) {
-	run(&testingBWrapper{b: b}, setup, env, "", false)
-}
-
-// RunCoprocess executes the benchmark by launching the Mixer through another process. The process is located
-// through an iterative search, starting with the current working directory, and using the executablePathSuffix as the
-// search suffix for executable (e.g. "bazel-bin/mixer/test/perf/perfclient/perfclient").
-func RunCoprocess(b *testing.B, setup *Setup, env *Env, executablePathSuffix string) {
-	run(&testingBWrapper{b: b}, setup, env, executablePathSuffix, true)
+// Run executes the benchmark using specified setup and settings.
+func Run(b *testing.B, setup *Setup, settings Settings) {
+	switch settings.RunMode {
+	case InProcess:
+		run(&testingBWrapper{b}, setup, &settings, false)
+	case CoProcess:
+		run(&testingBWrapper{b}, setup, &settings, true)
+	case DispatcherOnly:
```

Comment:
Can we perhaps make this a bit more generic? At least in its name?

Specifically, the interesting bit here is "local with no RPC overhead" right? That is, the point is not to interface with the dispatcher directly, the point is about bypassing gRPC.